resources/standards-data/langtags/langtags.json

```
@@ -93906,7 +94074,7 @@
         "full": "pht-Thai-TH",
         "iana": [ "Phu Thai" ],
         "iso639_3": "pht",
-        "latnnames": [ "Phu Thai or Phuu Thai", "Phu Thai or Phuu Thai" ],
+        "latnnames": [ "Phu Thai", "Phuu Thai" ],
```

```suggestion
        "latnnames": [ "Phu Thai or Phuu Thai", "Phu Thai or Phuu Thai" ],
```

Per discussion in https://github.com/silnrsi/langtags/issues/17#issuecomment-2097803177, this is valid.